osf/management/commands/poke_metrics_timespan_queries.py

```diff
@@ -0,0 +1,88 @@
+"""osf/management/commands/poke_metrics_timespan_queries.py
```

Comment:
I think it might be handy for generating test data on local devs.  It's a pain to generate a broad swath of pageview data locally.  Perhaps it should be renamed to reflect that, though.